test/e2e_test.go

```diff
@@ -223,8 +223,11 @@ livez check passed
 		}
 	})
 	It("exposes prometheus metrics", func() {
+
 		msPods := mustGetMetricsServerPods(client)
 		for _, pod := range msPods {
+			_, err := proxyRequestToPod(restConfig, pod.Namespace, pod.Name, "https", 4443, "/apis/metrics.k8s.io/v1beta1/")
```

Comment:
Please leave a comment that explains what this code does, it ensures that every MS instance get an requests so they expose all apiserver metrics.